src/Neo.CLI/Settings.cs

```diff
@@ -138,4 +141,21 @@ public ContractsSettings(IConfigurationSection section)
             }
         }
     }
+
+    public class PluginsSettings
+    {
+        public Uri DownloadUrl { get; } = new("https://api.github.com/repos/neo-project/neo-modules/releases");
+        public bool Prerelease { get; } = false;
+        public Version Version { get; } = Assembly.GetExecutingAssembly().GetName().Version!;
+
+        public PluginsSettings(IConfigurationSection section)
+        {
+            if (section.Exists())
+            {
+                DownloadUrl = section.GetValue(nameof(DownloadUrl), DownloadUrl)!;
+                Prerelease = section.GetValue(nameof(Prerelease), Prerelease);
+                Version = section.GetValue(nameof(Version), Version)!;
```

Comment:
This will force to update the config in each new version, currently is not required

Reply:
Its not required. It gets latest! It clear in the 1st comment of the PR. Its `optional` it you want to use a different version. That is why the configuration files don't have those options in there. Look at the `*.config` files.

Reply:
Why we would like to use 3.6.5 modules on 3.6.6 cli?

Reply:
if one has a bug or if one doesn't. Just because with every version most plugins don't change. Doesn't mean that you shouldn't be able to change the version. This is for development or testing. Which is why its removed in the config files.

Reply:
Why would you can the `DownloadUrl` to be configurable than? Answering that question will solve your problem.

Reply:
> Why would you want the `DownloadUrl` to be configurable than? Answering that question will solve your problem.

Not the same, and not required for me. If it has a bug and you want to downgrade the version, you should downgrade the cli also.

Reply:
What about testing?

Not required? Please remember what you said. https://github.com/neo-project/neo/pull/3080#issuecomment-1887286381

Reply:
Is good but not required, the plugins directory is not bad in the config, but the version? We don't needed and only can produce more confusion, that's my opinion